code/modules/admin/verbs/greytide_worldwide.dm

```diff
@@ -0,0 +1,152 @@
+/datum/greytide_worldwide/proc/event_setup_start()
+	return 1
+
+//Used to retrieve the list of mobs to convert. Called on each mob in eligible_mobs. Any mob this returns true for is added to the final conversion list.
+/datum/greytide_worldwide/proc/check_eligibility(var/mob/M)
+	if(M.stat == DEAD)
+		return 0
+	if(ishuman(M) || issilicon(M))
+		return 1
+
+//Does the actual converting of mobs. Called on each mob in the final conversion list. Returns the converted mob, in case conversion requires spawning a new one.
+//The base version of this proc turns just turns silicons into humans, so any events that require that need only call the parent and convert the mob returned.
+/datum/greytide_worldwide/proc/convert_mob(var/mob/M)
+	if(issilicon(M))
+		var/mob/living/silicon/S = M
+		var/mob/living/carbon/human/new_human = new /mob/living/carbon/human(S.loc, delay_ready_dna=1)
+		new_human.setGender(pick(MALE, FEMALE)) //The new human's gender will be random
+		var/datum/preferences/A = new()	//Randomize appearance for the human
+		A.randomize_appearance_for(new_human)
+		new_human.generate_name()
+		new_human.languages |= S.languages
+		if(S.default_language)
+			new_human.default_language = S.default_language
+		if(S.mind)
+			S.mind.transfer_to(new_human)
+		else
+			new_human.key = S.key
+		qdel(S)
+		M = new_human
+	return M
+
+//Called at the end of the event setup. Provided with the list of converted mobs in case something has to be done with them after everything else is done.
+/datum/greytide_worldwide/proc/event_setup_end(var/list/converted_mobs)
+
+
+//The classic.
+/datum/greytide_worldwide/greytide
+	name = "Greytide Worldwide"
+
+/datum/greytide_worldwide/greytide/check_eligibility(var/mob/M)
+	if(!..())
+		return 0
+	if(is_special_character(M))
+		return 0
+	return 1
+
+/datum/greytide_worldwide/greytide/convert_mob(var/mob/M)
+	var/mob/living/carbon/human/H = ..(M)
+
+	ticker.mode.traitors += H.mind
+	H.mind.special_role = GREYTIDE // NEEDED FOR CHEAT CHECKS!
+
+	H.mutations.Add(M_HULK) //all greytiders are permahulks
+	H.set_species("Human", force_organs=TRUE) // No Dionae
+	H.a_intent = I_HURT
+
+	H.update_mutations()
+	H.update_body()
+
+	/* This never worked.
+	var/datum/objective/steal/steal_objective = new
+	steal_objective.owner = H.mind
+	steal_objective.set_target("nuclear authentication disk")
+	H.mind.objectives += steal_objective
+	*/
+
+	var/datum/objective/hijack/hijack_objective = new
+	hijack_objective.owner = H.mind
+	H.mind.objectives += hijack_objective
+
+	to_chat(H, "<B>You are a greytider!</B>")
+	var/obj_count = 1
+	for(var/datum/objective/OBJ in H.mind.objectives)
+		to_chat(H, "<B>Objective #[obj_count++]</B>: [OBJ.explanation_text]")
+
+	for (var/obj/item/I in H)
+		if (istype(I, /obj/item/weapon/implant))
+			continue
+		if(isplasmaman(H)) //Plasmamen don't lose their plasma gear since they need it to live.
```

Comment:
convert_mob() only creates regular humans, why are you bothering with pointless checks?

Reply:
do i look like a coder to you

Reply:
You certainly don't.